.gitignore

```diff
@@ -1,3 +1,5 @@
+lib
```

Comment:
`tsconfig.json` defines an `outLib`, which is set to `./lib`. It's usually named the `dist` or `build` folder, but I've seen also `lib` for libraries. Felt like the more explicit choice.

Reply:
`dist` and `build` is what i saw most of the time. do you have some examples of projects using `lib`? currently i'm gravitating more towards `dist` or `build`

Reply:
for example https://basarat.gitbooks.io/typescript/docs/quick/library.html , besides the official docs the go-to guide

Reply:
and there are more articles suggesting the same https://www.google.com/search?ei=N-KNXaDMOMK4kwXXupjwBQ&q=lib+folder+typescript+library&oq=lib+folder+typescript+library&gs_l=psy-ab.3..0i71l8.0.0..5849...0.2..0.0.0.......0......gws-wiz.Nmxfz6dBQs0&ved=0ahUKEwjgw8fI4_DkAhVC3KQKHVcdBl4Q4dUDCAs&uact=5

As i said, `lib/` seemed good because it's explicit

Reply:
ok, convinced, thx for the links 👍